traitsui/testing/tester/_ui_tester_register/default_registry.py

```diff
@@ -31,6 +31,7 @@ def get_default_registry():
         return TargetRegistry()
     else:
         toolkit = {'wx': 'wx', 'qt4': 'qt4', 'qt': 'qt4'}[ETSConfig.toolkit]
-        module = importlib.import_module(".default_registry",
-                                         "traitsui.testing.tester." + toolkit)
+        module = importlib.import_module(
+            ".default_registry",
+            "traitsui.testing.tester._ui_tester_register." + toolkit)
```

Comment:
Minor: This would work too, which would have been immune to module name change:
```suggestion
            ".".join([__name__, toolkit])
```

Reply:
The CI is failing with:
```
File "/Users/appveyor/.edm/envs/traitsui-test-3.6-pyqt/lib/python3.6/site-packages/traitsui/testing/tester/_ui_tester_register/default_registry.py", line 36, in get_default_registry
    ".".join([__name__, toolkit]))
  File "/Users/appveyor/.edm/envs/traitsui-test-3.6-pyqt/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 941, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 950, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'traitsui.testing.tester._ui_tester_register.default_registry.qt4'; 'traitsui.testing.tester._ui_tester_register.default_registry' is not a package
```
as __name__ is including `.default_registry` before toolkit.  There is probably an easy solution to this, but I think for now I will just undo this change and merge this PR.  That change can be made in a simple separate PR

Reply:
Right, that needs to be the package, not the module
```
this_package, _ = __name__.rsplit(".", 1)
```
Then the `this_package` should be `"traitsui.testing.tester._ui_tester_register"`.

Reply:
>  There is probably an easy solution to this, but I think for now I will just undo this change and merge this PR. That change can be made in a simple separate PR

SGTM.